File: papers/ecoop2012/fetchTweets.scala
def fetchTweets(username: Rep[String]) = (ajax.get {
  new JSLiteral {
    val url = 
"http://api.twitter.com/1/statuses/user_timeline.json"
    val `type` = "GET"
    val dataType = "jsonp"
    val data = new JSLiteral {
      val screen_name = user
      val include_rts = true
      val count = 5
      val include_entities = true
    }
  }
}).as[TwitterResponse]

type TwitterResponse = Array[JSLiteral {val text: String}]

File: papers/ecoop2012/fetchTweets.js
function fetchTweets(username, callback) {
  jQuery.ajax({
    url: "http://api.twitter.com/1/
      statuses/user_timeline.json/",
    type: "GET",
    dataType: "jsonp",
    data: {
      screen_name : username,
      include_rts : true,
      count : 5,
      include_entities : true
    },
    success: callback
  })
}
